corehq/apps/userreports/reports/filters/values.py

```diff
@@ -249,6 +255,11 @@ def to_sql_filter(self):
                 self.filter['field'],
                 get_INFilter_bindparams(self.filter['slug'], ['start_date', 'end_date'])
             )
+        elif self._is_empty():
+            return ORFilter([
+                EQFilter(self.filter['field'], self.filter['slug']),
+                ISNULLFilter(self.filter['field']),
+            ])
         elif self._is_null():
             return self._null_filter(self.filter['field'])
```

Comment:
One thing I debated was changing the behavior of `_is_null` to also be the same as the above. Then any prefilters that have a value of `null` would also include empty strings. It'd be easy to change here, though would trip up someone explicitly wanting to differentiate between those two values. Is that ever a valid need? 

I didn't do this just to avoid controversy / potentially breaking changes, but I think it is more user-friendly behavior in all scenarios I can think of.

Not sure if someone on @dimagi/product can weigh in on that.

Reply:
We can discuss on the proposed change here: https://github.com/dimagi/commcare-hq/pull/28315